src/stamps.ts

```diff
@@ -148,6 +166,10 @@ export async function topUpStamp(beeDebug: BeeDebug, postageBatchId: string, amo
   return stamp
 }
 
+export async function diluteStamp(beeDebug: BeeDebug, postageBatchId: string, depth: number) {
```

Comment:
I would argue that this function is not needed. You can call the `diluteBatch` on `beeDebug` directly...